packages/docs/docs/concepts/progressive-loading/stackProgressive.md

```diff
@@ -0,0 +1,78 @@
+---
+id: stackProgressive
+---
+
+# Stack Progressive Loading
+
+Images for the stack viewport can be loaded with a lower resolution/lossy
+version first, followed by increasingly higher resolutions, and finally
+the final version being a lossless representation.
+
+For HTJ2K, this is done automatically when the image is encoded in progressive
+resolution order by using a streaming reader that returns lower resolution versions
+of the image as they are available.
+
+For other image types, a separate lower resolution/lossy version is required.
+The Static DICOMweb toolkit includes some options to create such images.
+
+# Performance
+
+In general, about 1/16-1/10th of the image is retrieved for the lossy/first
+version of the image. This results in about a 10x speed improvement to first
+images. It is affected fairly strongly by overall image size, network performance
+and compression ratios.
+
+| Type             | Network | First Render | Final Render |
+| ---------------- | ------- | ------------ | ------------ |
+| JLS              | 4g      |              | 4586 ms      |
+| JLS Reduced      | 4g      | 359 ms       | 4903 ms      |
+| HTJ2K            | 4g      | 897 ms       | 5213 ms      |
+| HTJ2K Byte Range | 4g      | 780 ms       | 5431 ms      |
+
+- JLS Reduced uses 1/16 size JLS 'thumbnails'
+- HTJ2K uses streaming data
+- HTJ2K Byte Range uses 64k initial retrieve, followed by remaining data
+
+# Configuration
+
+See the stackProgressive example for stack details.
+
+No special configuration is required for HTJ2K images, however they can be
+disabled by setting the streaming values to false, for example:
+
+```javascript
+cornerstoneDicomImageLoader.configure({
+  retrieveOptions: {
+      default: {
+        '3.2.840.10008.1.2.4.96': {
+        // Need a specific streaming to false to disable streaming support
+        streaming: false,
+```
```

Comment:
This is just a confiugration, it is not the load actually, can you add the example of how to load using this configuration after?

Reply:
we should explain that stack viewport uses the singleFast and singleFinal (I still need to understand what they mean, we can talk in our meeting) by default, and this is how you should change that ...